testing/matrix-sdk-integration-testing/src/tests/sliding_sync/room.rs

```diff
@@ -348,17 +344,16 @@ async fn test_room_notification_count() -> Result<()> {
     loop {
         assert!(info_updates.next().await.is_some());
 
-        let count = alice_room.unread_notification_counts();
-        if count == prev_count {
-            // Sometimes we get notified for changes to unrelated, other fields of
-            // `info_updates`.
+        // FIXME we receive multiple spurious room info updates.
```

Comment:
Why does it happen?

Reply:
I think `set_latest_event` causes a room info update, and then later we save the room info in the state changes and apply_changes will set the room info again. I've fixed one issue like that in a previous PR, but didn't get to fix all of them yet. It's only a spurious issue causing a bit of overhead for room info subscribers. We could fix this later.